templates/mustache-templates/teamsBot.js.mustache

```diff
@@ -2,55 +2,63 @@
 const axios = require("axios");
 const querystring = require("querystring");
 {{/IS_ME}}
-const { TeamsActivityHandler, CardFactory{{#IS_BOT}}, ActionTypes{{/IS_BOT}}} = require("botbuilder");
+const { TeamsActivityHandler, CardFactory{{#IS_BOT}}, TurnContext{{/IS_BOT}}} = require("botbuilder");
```

Comment:
Can this simplified bot template also work well when user choose both Bot and ME capabilities?

Reply:
Yes